plugins/statistics.koplugin/main.lua

```diff
@@ -559,15 +559,15 @@ function ReaderStatistics:updateSettings()
         fields = {
             {
                 text = "",
```

Comment:
I think you could use `text = self.page_min_read_sec,`(what you put into `hint`), and it should be there as pre-filled value and editable, and you'll get it back if the user applies changes.